app/src/test/java/ch/epfl/sdp/blindly/unit_tests/AudioRecordTest.kt

```diff
@@ -1,8 +1,7 @@
-package ch.epfl.sdp.blindly.recording
+package ch.epfl.sdp.blindly.unit_tests
```

Comment:
I don't think the new package name is relevant, as Tests under test/ are implicitly unit tests. It seems redundant to me.